test/src/094-fuser/main

```diff
@@ -0,0 +1,45 @@
+
+cvmfs_test_name="Using cvmfs_config fuser to find processes accessing a repository"
+cvmfs_test_suites="quick"
+
+cvmfs_run_test() {
+  logfile=$1
+
+  echo "Mount repository"
+  cvmfs_mount grid.cern.ch || return 1
+
+  echo "Start background process accessing it and check that it is being used"
+  set -m # enables job control handling
+  (cd /cvmfs/grid.cern.ch; sleep 30) &
```

Comment:
```suggestion
  (cd /cvmfs/grid.cern.ch; sleep 10000) &
```
I think we should wait essentially infinitely. This should make the test less flaky and in case of a hang the test driver timeout cleans up.